app/views/stafftools/impersonate.js.erb

```diff
@@ -0,0 +1 @@
+$('#js-user-search-results').html("<%= escape_javascript(render partial: 'stafftools/impersonate/user_results', locals: { users: @users }) %>");
```

Comment:
it's not really loading any scripts all this is doing it replacing the partial with the new set of results.

This is the partial https://github.com/education/classroom/pull/359/files#diff-0eb563d2788c58b421c0979a6f4a4c04R1